externals/kyuubi-spark-sql-engine/src/main/resources/python/execute_python.py

```diff
@@ -135,11 +172,10 @@ def parse_code_into_nodes(code):
 
         # Convert the chunks into AST nodes. Let exceptions propagate.
         for chunk in chunks:
-            # TODO: look back here when Jupyter and sparkmagic are supported
-            # if chunk.startswith('%'):
-            #     nodes.append(MagicNode(chunk))
```

Comment:
add magic node back

Reply:
Thanks for bringing back this feature. 
Could you attach some screenshots or response outputs for this ?

Reply:
updated the pr description.